src/components/tags/TagProvider.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from "react"
+export const TagContext = React.createContext()
+
+export const TagProvider = (props) => {
+    const [tags, setTags] = useState([])
+
+    const get_all_tags = () => {
```

Comment:
should this function be named getTags to stay with the naming conventions in the other providers on the front-end?